### ISSUE TITLE
[BITV] 9.2.4.3/3.10 - Keep focus on selected option in submenus

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/text#3864]
- [ ] Fix
- [ ] Backport

When text is selected and a style "heading" has been set and the the headings menu is opened again, the focus should be on the selected option, which is just visually indicated. In addition to visually highlighting the selected option, application menus should use the aria-activedecendant Attribute to reflect the current focus to the currently active option within the menu once it is opened. It should also support the recommended keyboard navigation pattern (arrow key navigation within toolbar). Arrow key navigation in menu is supported but does not cycle through menu options. Currently this is difficult to evaluate properly due to the rapid automatic closing of the popup menus and the VERY sluggish updating when using keyboard navigation. Same thing affects popup insert emois. (2)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v3-n10

</details>

- [ ] 9.2.4.3/10.10 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v10-n10

**Comment by juliushaertl** [comment]
Pushed a proposal to fix this to https://github.com/nextcloud-libraries/nextcloud-vue/pull/4678

Not fully happy but the best I could figure out for now considering Monday as a target date to get this in.
